ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -36,6 +36,65 @@ ethernet_interfaces:
     switchport:
       port_security:
         enabled: true
+      enabled: false
```

Comment:
if possible please improve the coverage for only switchport option in ethernet and port-channel modules. Let me know if you need any help on the same. I see there are few test missing which can improve the coverage